docs/content/using/changing_host_connectivity.md

```diff
@@ -0,0 +1,18 @@
+# Changing host connectivity
+
+Host connectivity for dynamic host definition needs to be updated when node connectivity changes take place.
+
+Before you begin, ensure that the old host definition is deleted.
+
+Use the following procedure to redefine host connectivity:
+
+1. Undeploy the CSI node pod from the relevant node that the host definition is a part of.
+2. Verify that all host definition instances of the node are deleted.
+     
+          $> kubectl get hostdefinition | grep <hostname> | wc -l
```

Comment:
Should be - `<nodename>`

Reply:
replaced across files

Reply:
Can be `<node-name>` I wanted to be clear it is not just the hostname of the node but from the k8s cluster perspective, its nodes.

Reply:
LGTM. will let @roysahar resolve when they see fit